packages/orchestration/test/examples/send-cctp.test.ts

```diff
@@ -0,0 +1,243 @@
+      },
+      transferChannel: {
+        counterPartyChannelId: 'channel-10062',
+        channelId: 'channel-65',
+        ...txChannelDefaults,
+      },
+    } as IBCConnectionInfo,
+  },
+};
+
+const registerUSDC = async ({ bankManager, agoricNamesAdmin }) => {
+  const usdcKit = withAmountUtils(makeIssuerKit('USDC'));
+  const { issuer, mint, brand } = usdcKit;
+
+  // TODO: use USDC denom
+  // const { axelar } = AgoricDevnetConfig.chains;
+  const { noble } = fetchedChainInfo;
+  const { channelId: agoricToNoble } =
+    fetchedChainInfo.agoric.connections[noble.chainId].transferChannel;
+  const denom = `ibc/${denomHash({ channelId: agoricToNoble, denom: 'uusdc' })}`;
+
+  const issuerName = 'USDC';
+  const proposedName = 'USDC noble';
+  await E(bankManager).addAsset(denom, issuerName, proposedName, {
+    issuer,
+    mint,
+    brand,
+  });
+  await E(E(agoricNamesAdmin).lookupAdmin('vbankAsset')).update(
+    denom,
+    /** @type {AssetInfo} */ harden({
+      brand,
+      issuer,
+      issuerName,
+      denom,
+      proposedName,
+      displayInfo: { IOU: true },
+    }),
+  );
+
+  return harden({ ...usdcKit, denom });
+};
+
+test('send to base via noble CCTP', async t => {
+  t.log('bootstrap, orchestration core-eval');
+  const {
+    bootstrap,
+    commonPrivateArgs,
+    brands: { ist },
+    utils: { inspectLocalBridge, transmitTransferAck, populateChainHub },
+    mocks: { ibcBridge },
+  } = await commonSetup(t);
+  const vt = bootstrap.vowTools;
+
+  populateChainHub();
+  const { zoe, bundleAndInstall } = await setUpZoeForTest();
+
+  t.log('contract coreEval', contractName);
+
+  const installation: Installation<StartFn> =
+    await bundleAndInstall(contractFile);
+
+  const { bankManager, agoricNamesAdmin } = bootstrap;
+  const usdcKit = await registerUSDC({
+    bankManager,
+    agoricNamesAdmin,
+  });
+
+  const storageNode = await E(bootstrap.storage.rootNode).makeChildNode(
+    contractName,
+  );
+
+  // Mix osmosis from AgoricDevnetConfig with commonPrivateArgs
+  const {
+    agoric,
+    osmosis: _x,
+    ...withoutOsmosis
+  } = commonPrivateArgs.chainInfo;
+  const { osmosis } = AgoricDevnetConfig.chains;
+  const connections = {
+    ...agoric.connections,
+    ...AgoricDevnetConfig.connections,
+    // XXX to get from Agoric to axelar, use osmosis
+    'axelar-testnet-lisbon-3': AgoricDevnetConfig.connections[osmosis.chainId],
+  };
+  const chainInfo = {
+    ...withoutOsmosis,
+    osmosis,
+    agoric: { ...agoric, connections },
+  };
+  const assetInfo: Array<[string, DenomDetail]> = [
+    ...commonPrivateArgs.assetInfo,
+  ];
+  const sendKit = await E(zoe).startInstance(
+    installation,
+    { Stable: ist.issuer, USDC: usdcKit.issuer },
+    {},
+    {
+      ...commonPrivateArgs,
+      assetInfo,
+      chainInfo,
+      storageNode,
+    },
+  );
+
+  t.log('register the base chain');
+  await E(sendKit.creatorFacet).registerChain(
+    'base',
+    {
+      chainId: 'E8453',
```

Comment:
```suggestion
      chainId: '8453',
```
Test only, but let's still use the actual value